### ISSUE TITLE
Tooltips - Mouse Over Tooltip Description dissapears too fast

### ISSUE THREAD

**Issue by PulsarNeutronStar** [soda3x/ArmaReforgerServerTool#46]
The Tooltips / Descriptions that appear while hovering the mouse over the text of the setting is a godsent.  Great feature!

But sometimes, the tooltip disappears rather quickly, even when i am not touching the mouse. 

Maybe there is some jitter on my input (also, im using remote desktop) but would it be possible to "optimize" the tooltip, and only have it disappear when the mouse leaves the area above the text instead of mouse movement?

**Comment by soda3x** [comment]
Interesting, I'll look into it, for now, trying hovering over the empty space next to the name of the field. I have an inkling that it might be due to the fact that the label, the input and the background all have the same tooltip associated with them.

**Comment by PulsarNeutronStar** [comment]
https://streamable.com/qz2107

**Comment by soda3x** [comment]
> https://streamable.com/qz2107

Yep, this behaviour is caused by overlapping hitboxes. I'll try and fix it for the next patch. Sorry!

**Comment by PulsarNeutronStar** [comment]
> > https://streamable.com/qz2107
> 
> Yep, this behaviour is caused by overlapping hitboxes. I'll try and fix it for the next patch. Sorry!

all good. software still works so its nothing critical, just a QoL thing :D

Thanks for making reforger hosting a breeze with your tool :)

**Comment by soda3x** [comment]
Finally fixed!
